specification/imagebuilder/resource-manager/Microsoft.VirtualMachineImages/preview/2019-05-01-preview/examples/ListRunOutputs.json

```diff
@@ -2,12 +2,12 @@
   "parameters": {
     "subscriptionId": "{subscription-id}",
     "resourceGroupName": "myResourceGroup",
-    "imageTemplateName": "myImageTemplate",
+    "imageTemplateName":"myImageTemplate",
```

Comment:
Previous formatting in this file was correct

Reply:
fixed this and the others